.chloggen/azuremonitorexporter-fix-span-attributes-in-properties.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
This is now a breaking change I guess?

Reply:
Changed the bug type to braking - https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29684/commits/f58089d678ea8df3a20e06150ee9eb2d7aaa355d